designer/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/BaseEndpointDefinitions.scala

```diff
@@ -67,37 +66,20 @@ object BaseEndpointDefinitions {
                   )
                 )
             ),
-            oneOfVariantFromMatchType(
-              NotFound,
-              plainBody[ImpersonatedUserDataNotFoundError.type]
-                .description("Identity provided in the Nu-Impersonate-User-Identity header did not match any user")
-                .example(
-                  Example.of(
-                    summary = Some("No impersonated user's data found for provided identity"),
-                    value = ImpersonatedUserDataNotFoundError
-                  )
-                )
-            ),
             oneOfVariantFromMatchType(
               Forbidden,
-              plainBody[ImpersonationMissingPermissionError.type]
-                .example(
-                  Example.of(
-                    summary = Some("Authorization failed, user does not have permission to impersonate"),
-                    value = ImpersonationMissingPermissionError
+              oneOfBody[SecurityError](
```

Comment:
Shouldn't we have `AuthorizationError` here instead?

Reply:
If we would have here only `InsufficientPermission` and `ImpersonationMissingPermissionError` they yes - but  we also have `ImpersonatedUserDataNotFoundError` in here

Reply:
ok, but can we make "ImpersonatedUserDataNotFoundError" an AuthorizationError? You proposed [here](https://github.com/TouK/nussknacker/pull/7014#discussion_r1802585801) to interpret it as 403

Reply:
Yes we could, then we could change that - good point.

Reply:
Done + moved `ImpersonatedUserNotExistsError` to `401` section, since based on created model it should be there - it is part of AuthenticationError rather than AuthorizationError.